eng/packaging.targets

```diff
@@ -219,15 +220,20 @@
       <None Include="$(_NETStandardCompatErrorFilePath)"
             PackagePath="buildTransitive\%(NETStandardCompatError.Identity)"
             Pack="true" />
+      <!-- Add the placeholder file for the supported target framework group if that directory is empty. -->
       <None Include="$(PlaceholderFile)"
-            PackagePath="buildTransitive\%(NETStandardCompatError.Supported)"
-            Pack="true" />
+            PackagePath="$(_NETStandardCompatErrorPlaceholderFilePackagePath)"
+            Pack="true"
+            Condition="('@(Content)' == '' or
+                        !@(Content->AnyHaveMetadataValue('PackagePath', '$(_NETStandardCompatErrorPlaceholderFilePackagePath)'))) and
+                       ('None)' == '' or
+                        !@(None->AnyHaveMetadataValue('PackagePath', '$(_NETStandardCompatErrorPlaceholderFilePackagePath)')))" />
```

Comment:
This assumes the `PackagePath` is set to the directory. But there can be cases where it is the full file name. For example:

[https://github.com/dotnet/runtime/blob/a31ed471c8f528fe7ae0237ba38d74ea9af7d0c5/eng/packaging.targets#L156-L158](https://github.com/dotnet/runtime/blob/a31ed471c8f528fe7ae0237ba38d74ea9af7d0c5/eng/packaging.targets#L156-L158)

Do we want to handle that scenario here?

Reply:
Unfortunately that isn't possible with the current set of item functions. There is no AnyHaveMetadataContainsValue() function. It would be possible but would make the code much much bigger.

But let me try that out offline.

Reply:
Fixed.